### DISCUSSION TITLE
Why it takes so long tot import?

### DISCUSSION THREAD

**Discussion by RaresAil** [gerbera/gerbera#2631] [Q&A] [answered]
What is the import for? For SQLite or MySQL, the import it takes a lot, what it is doing that is taking so long?

**Comment by KarlStraussberger** [comment, marked as answer]
The import is ran only once. It reads all the metadata and builds up the virtual tree based on the metadata. When you browse or search your media library the files are not accessed.

Speed always depends on many factors: disk speed, main memory, processor.

My 10 year old Phemon does 30000 music tracks, 5000 videos and 100000 images in two hours. That's fast for me.
